pkg/tests/end_to_end_tests/multi_tenancy_test.go

```diff
@@ -736,6 +736,99 @@ func TestMultiTenancyWithValidTenantsAsLabels(t *testing.T) {
 	})
 }
 
+func TestMultiTenancyLabelNames(t *testing.T) {
+	ts, _ := generateSmallMultiTenantTimeseries()
+	withDB(t, *testDatabase, func(db *pgxpool.Pool, t testing.TB) {
+		getClient := func(auth tenancy.Authorizer) *pgclient.Client {
```

Comment:
Is it possible to convert this into test table ? I see quite a bit redundancy and might possible to generalise into TT.